site/docs/samples/omd-discover-s3-asset.md

```diff
@@ -0,0 +1,61 @@
+# Discovering an S3 asset through the OpenMetadata UI
+
+This page explains how to discover an existing S3 asset through the OpenMetadata UI. The screenshots refer to `localstack` cloud storage, but the explanations also apply to other S3 services.
+
+Begin by opening your browser to the OpenMetadata UI. If you installed OpenMetadata in your kubernetes cluster in the `open-metadata` namespace, go to http://localhost:8585 after running:
+```
```

Comment:
in order to make http://localhost:8585 a link when running in the site,  wrap it in [ ] and add ( link )